### ISSUE TITLE
[Discussion] Removal of Prerelease .NET Core 1.x SDKs and runtimes from App Service

### ISSUE THREAD

**Issue by davidebbo** [Azure/app-service-announcements-discussions#7]
Discussion thread for https://github.com/Azure/app-service-announcements/issues/27

**Comment by hajekj** [comment]
Is there some official list of current .NET Core SDKs available on App Service? I couldn't really find any...

**Comment by davidebbo** [comment]
Here is the full list:

- 1.0.0-preview2.1-003155
- 1.0.0-preview2-003156
- 1.0.0-preview2-1-003177
- 1.0.0-rc3-004530
- 1.0.0-rc4-004771
- 1.0.1
- 1.0.4
- 2.0.0-preview1-005977
- 2.0.0-preview2-006497

And soon, we'll have 2.0.

Note that the design of the SDK is such that the newest version can be used for any official runtime version. So once we have 2.0, it can be used for all apps.

In fact, apps that don't have a global.json (which is most apps since VS & `dotnet new` don't  create one) automatically use the latest installed SDK. So today, almost everyone is already using `2.0.0-preview2-006497` when building on Antares (i.e. Kudu scenario).

Bottom line is that for the SDKs, we really just need to keep the latest on the box.

**Comment by hajekj** [comment]
Thanks for the answer. We are hitting some package restore errors from MyGet feed while it builds fine (with warning only) through 1.0.4, so just wanted to make sure that it is here to stay for a while.

**Comment by davidebbo** [comment]
Yes, I think we'll keep 1.0.4 for a while as a backup. But once 2.0 comes out (soon!), if it still can't build your app, we need to discuss with Core team, as their claim is that it can build all 1.x apps.
